src/test/java/Homework23.java

```diff
@@ -0,0 +1,21 @@
+import PageFactory.HomePage;
+import PageFactory.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework23 extends BaseTest {
+
+    @Test
+    public void renamePlaylist() {
+        String playlistName = "PlayMusic";
```

Comment:
The notification message is displayed only when the playlist name is updated to new. Using the same String name multiple times does not make the system display a notification alert. This always leads test to be failed. To make assertion work you either need to change the String value in each test run or else could use the "Random" class to generate a random name each time.
`Random random = new Random();`
        `String playlistName = random.toString();`